docs/guides/storage-profiles/readme.md

```diff
@@ -232,6 +232,55 @@ data:
 [...]
 ```
 
+### Target Layout
+
+Users may want Lustre file systems with different performance characteristics. For example, a user job with a single compute node accessing the Lustre file system would see acceptable performance from a single OSS. A FPP workload might want as many OSSs as posible to avoid contention.
```

Comment:
```suggestion
Users may want Lustre file systems with different performance characteristics. For example, a user job with a single compute node accessing the Lustre file system would see acceptable performance from a single OSS. An FPP workload might want as many OSSs as possible to avoid contention.
```